internal/cli/install.go

```diff
@@ -499,12 +498,8 @@ func installRunner(
 		AuthToken:     resp.Token,
 	}
 
-	// Get ID for runner
-	id, err := server.Id()
-	if err != nil {
-		ui.Output("Error generating runner ID: %s", clierrors.Humanize(err), terminal.WithErrorStyle())
-		return 1
-	}
+	// We set the ID to be "init" since it is the initial static runner
+	id := "init"
```

Comment:
Maybe a nitpick but I'm wondering if we should somehow indicate this is a runner since just listing helm deployments shows the default runner as `waypoint-init`, maybe the k8s runner installer can use `waypoint-runner-%s` instead of just `waypoint-%s` or just a simple `waypoint-runner` for the initial runner.

May make the deployment a bit clearer to an operator after they have forgotten about this in the future after an initial installation.

Reply:
Good catch! I missed that here:

https://github.com/hashicorp/waypoint/blob/bf26572071b113d5a24ae6fb7605363a4a24ae7b/internal/runnerinstall/k8s.go#L62

Updating to prefix the runner's ID with `waypoint-runner-` instead!